Library/Homebrew/cmd/tap.rb

```diff
@@ -15,7 +15,7 @@ def tap
     else
       user, repo = tap_args
       clone_target = ARGV.named[1]
-      opoo "Already tapped!" unless install_tap(user, repo, clone_target)
+      opoo "#{user}/#{repo} already tapped" unless install_tap(user, repo, clone_target)
```

Comment:
It'd be nice to keep the `!` or even a `.` at the end of the sentence, but that nit aside I'm a big :+1: on this change, personally.